ansible_base/authentication/utils/claims.py

```diff
@@ -3,20 +3,18 @@
 from typing import Optional
 
 from django.conf import settings
-from django.contrib.auth import get_user_model
+from django.contrib.auth.models import AbstractUser
 from django.utils.timezone import now
 from rest_framework.serializers import DateTimeField
-from social_core.pipeline.user import get_username
 
-from ansible_base.authentication.models import Authenticator, AuthenticatorMap, AuthenticatorUser
-from ansible_base.authentication.social_auth import AuthenticatorStorage, AuthenticatorStrategy
+from ansible_base.authentication.models import Authenticator, AuthenticatorMap
 
 from .trigger_definition import TRIGGER_DEFINITION
 
 logger = logging.getLogger('ansible_base.authentication.utils.claims')
 
 
-def create_claims(authenticator: Authenticator, username: str, attrs: dict, groups: list) -> (bool, bool, dict, list):
+def create_claims(authenticator: Authenticator, username: str, attrs: dict, groups: list[str]) -> (bool, bool, dict, list):
```

Comment:
The return type annotation is incorrect. Tuple of types is invalid annotation in general and this function returns dictionary on line 84.